core/arch/arm/plat-sam/nsec-service/smc_ids.h

```diff
@@ -5,12 +5,19 @@
 
 #ifndef SMC_IDS_H
 #define SMC_IDS_H
+#include <optee_msg.h>
+#include <sm/optee_smc.h>
+
+#define SAMA5_SMC_SIP_SCMI_CALL_ID	0x200
 
 #define SAMA5_SMC_SIP_SFR_SET_USB_SUSPEND	0x300
 
 #define SAMA5_SMC_SIP_SET_SUSPEND_MODE	0x400
 #define SAMA5_SMC_SIP_GET_SUSPEND_MODE	0x401
 
+/* SCMI */
+#define SAMA5_SMC_SIP_SCMI_CALL_ID	0x200
```

Comment:
duplicated :)

Reply:
Indeed, will remove the duplicate for the first commit.